packages/lib/src/buildInfo.ts

```diff
@@ -21,20 +26,47 @@ export function registerBuildInfo() {
 
   const gauge = getMeter().createUpDownCounter("build_info");
   gauge.add(1, buildInfo);
+
+  return buildInfo;
 }
 
-function getVersion() {
-  if (process.env.npm_package_version) {
-    return process.env.npm_package_version;
+function getVersion(): string | undefined {
+  if (process) {
+    if (process.env.npm_package_version) {
+      return process.env.npm_package_version;
+    }
+    return process.env.PACKAGE_VERSION || process.env.AUTOMETRICS_VERSION;
+    //@ts-ignore
+  } else if (Deno) {
+    return (
+      //@ts-ignore
+      Deno.env.get("AUTOMETRICS_VERSION") || Deno.env.get("PACKAGE_VERSION")
+    );
+  } else {
+    return;
```

Comment:
No need for the `else if` here as you're already early returning above. You can omit the last `else` statement too as the function implicitly returns when none of the conditionals are met.